app/views/miq_request/_prov_field.html.haml

```diff
@@ -292,19 +292,18 @@
         -# tree control for tags fields
         .col-md-8
           #all_tags_treebox.treeview-pf-hover.treeview-pf-select{:style => "color:#000; overflow: hidden;"}
-          - if @edit && @edit[:req_id]
-            - check = @edit[:req_id]
-          - elsif @miq_request
-            - check = @miq_request.id
-          - else
-            - check = 'new'
-          = render(:partial => "layouts/tree",
-                   :locals => {:tree_id       => "all_tags_treebox",
-                               :tree_name     => "all_tags_tree",
-                               :bs_tree       => @all_tags_tree,
-                               :oncheck       => "miqOnCheckProvTags",
-                               :check_url     => "/miq_request/prov_field_changed/#{check}",
-                               :checkboxes    => true})
+            - if @edit && @edit[:req_id]
```

Comment:
(in shared/tree, the JS bits get rendered *inside* the tree element, not right after, so doing the same here)

Reply:
@PanSpagetka will be happy, AFAIK he's trying to get rid of this tree :laughing:

Reply:
Yes, I am trying to cut this tree down!

Reply:
Feel free to, I can skip the change if you're removing it.